cmd/sanitize-prow-jobs/main_test.go

```diff
@@ -19,7 +20,20 @@ func TestDefaultJobConfig(t *testing.T) {
 		Periodics: []config.Periodic{{}, {}, {JobBase: config.JobBase{Cluster: "default"}}},
 	}
 
-	defaultJobConfig(jc)
+	config := &Config{Clusters: struct {
+		Default    string `json:"default"`
```

Comment:
See and *this* is why anonymous structs have a very limited use-case...

Reply:
Done.